tests/dummy/app/pods/link/template.hbs

```diff
@@ -120,6 +120,25 @@
       </div>
     </div>
 
+      <div class="example">
+          <div class="title">in-line</div>
+          <div class="demo">
+            {{frost-link 'In-line' 'link.min'
```

Comment:
I'm having trouble understanding how these dummy app additions relate to the changes in the component.